tasks/src/main/java/taf/product/yandex/disk/page/YandexDiskAuthPage.java

```diff
@@ -20,6 +24,7 @@ public YandexDiskAuthPage(WebDriver driver) {
   }
 
   public YandexDiskAuthPage enterLogin(String login) {
+    logger.info("Type login in form from property");
```

Comment:
It will be useful to log a string that you type. Please fix in all similar places